lib/ExpensiMark.js

```diff
@@ -473,7 +474,8 @@ export default class ExpensiMark {
                 replacement: (match, g1, offset, string, extras) => {
                     const reportToNameMap = extras.reportIdToName;
                     if (!reportToNameMap || !reportToNameMap[g1]) {
-                        return '';
+                        Log.alert(`[ExpensiMark] Missing report name for ID: ${g1}`);
```

Comment:
Variable parameters should go in the 2nd parameter of the call, so:
```suggestion
                        Log.alert('[ExpensiMark] Missing report name', {reportID: g1});
```

Reply:
resolved 👌🏼